### PR TITLE
Expand list of entity type storages

### LINKED ISSUE #240: Expand list of entity type storages
I wanted to think of a great entry-level issue for someone new who wants to contribute to this project.

https://github.com/mglaman/phpstan-drupal/blob/main/extension.neon#L16-L19

I would love to expand the default list of known entity storages based on core entity types and major contrib.

All that needs to be done are additions to this map. For example, from this project's README:

```
        entityTypeStorageMapping:
            search_api_index: Drupal\search_api\Entity\SearchApiConfigEntityStorage
            search_api_server: Drupal\search_api\Entity\SearchApiConfigEntityStorage
```

### PR DESCRIPTION
**Pull request by ptt-homme**
Fix #240 & #271

### CODE REVIEW

**Review by mglaman — CHANGES_REQUESTED** [review]
This fixes the documentation but doesn't expand the definitions. I'll help push this over the edge using the script referenced in the issue.

**Comment by mglaman** [comment]
:O @ptt-homme thanks!

**Review by mglaman — APPROVED** [review]

**Comment by ptt-homme** [comment]
@mglaman I did this this week-end. I was not sure if it goes in the right direction and what is the list of major contrib expected. If it's ok, I'll add the rest by the end of the week.

**Comment by mglaman** [comment]
@ptt-homme this is amazing. I plan on merging during my live stream. I'll open some issues for follow-ups of major contribs that would be useful to add.

**Comment by ptt-homme** [comment]
Ok, I'll wait for the issues and propose a new PR with the rest then. I also have a PR ready which extract PHP actions into multiple jobs. Do you mind having a look at it ?

**Comment by mglaman** [comment]
@ptt-homme yeah, I'll open a new issue and you can make a PR for it. I will take a look at the actions PR during my live stream

File: README.md [hunk 1]
```diff
@@ -111,23 +111,36 @@ default mapping can be found in `extension.neon`. For example:
...
 ```
 parameters:
-    drupal:
-        entityTypeStorageMapping:
-            node: Drupal\node\NodeStorage
-            taxonomy_term: Drupal\taxonomy\TermStorage
-            user: Drupal\user\UserStorage
-            block: Drupal\Core\Config\Entity\ConfigEntityStorage
+	drupal:
+		entityTypeStorageMapping:
```
Comment (mglaman) [inline README.md:115]: s/entityTypeStorageMapping/entityMapping

File: README.md [hunk 2]
```diff
@@ -111,23 +111,36 @@ default mapping can be found in `extension.neon`. For example:
...
 example for adding a mapping for Search API:
 
 ```
 parameters:
-    drupal:
-        entityTypeStorageMapping:
-            search_api_index: Drupal\search_api\Entity\SearchApiConfigEntityStorage
-            search_api_server: Drupal\search_api\Entity\SearchApiConfigEntityStorage
+	drupal:
+		entityTypeStorageMapping:
```
Comment (mglaman) [inline README.md:136]: s/entityTypeStorageMapping/entityMapping

File: README.md [hunk 3]
```diff
@@ -145,11 +158,19 @@ The default mapping can be found in `extension.neon`:
 ```neon
 parameters:
 	drupal:
-		entityStorageMapping:
-			node: Drupal\node\Entity\Node
-			taxonomy_term: Drupal\taxonomy\Entity\Term
-			user: Drupal\user\Entity\User
-			block: Drupal\block\Entity\Block
+		entityTypeStorageMapping:
```
Comment (mglaman) [inline README.md:161]: s/entityTypeStorageMapping/entityMapping

**Pull request merged**